homes-ms-storesmanagement/src/main/java/com/tenx/ms/storesmanagement/orders/rest/dto/OrderDTO.java

```diff
@@ -0,0 +1,124 @@
+package com.tenx.ms.storesmanagement.orders.rest.dto;
+
+
+import com.tenx.ms.commons.validation.constraints.Email;
+import com.tenx.ms.commons.validation.constraints.PhoneNumber;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.List;
+
+
+public class OrderDTO {
+
```

Comment:
i do agree with you that the DTO qualifier is in the package name , but as usually you don't use the package name in most of the code (because you imported the type in the imports section) i believe that this will increase readability in the rest of the code